### ISSUE TITLE
Collector-specific flags missing

### ISSUE THREAD

**Issue by fischerman** [prometheus-community/windows_exporter#648]
The help section suggests, that the collector-specific flags are not available.
```
usage: windows_exporter [<flags>]

Flags:
  -h, --help                    Show context-sensitive help (also try
                                --help-long and --help-man).
      --config.file="C:\\Program Files\\windows_exporter\\config.yml"  
                                YAML configuration file to use. Values set in
                                this file will override flags.
      --telemetry.addr=":9182"  host:port for exporter.
      --telemetry.path="/metrics"  
                                URL path for surfacing collected metrics.
      --telemetry.max-requests=5  
                                Maximum number of concurrent requests. 0 to
                                disable.
      --collectors.enabled="cpu,cs,logical_disk,net,os,service,system,textfile"  
                                Comma-separated list of collectors to use. Use
                                '[defaults]' as a placeholder for all the
                                collectors enabled by default.
      --collectors.print        If true, print available collectors and exit.
      --scrape.timeout-margin=0.5  
                                Seconds to subtract from the timeout allowed by
                                the client. Tune to allow for overhead or high
                                loads.
      --log.level="info"        Only log messages with the given severity or
                                above. Valid levels: [debug, info, warn, error,
                                fatal]
      --log.format="logger:stderr"  
                                Set the log target and format. Example:
                                "logger:syslog?appname=bob&local=7" or
                                "logger:stdout?json=true"
      --version                 Show application version.
```
For examples:
```
C:\Users\Administrator\go\src\github.com\prometheus-community\windows_exporter>.\windows_exporter.exe --collectors.exchange.list
time="2020-11-02T15:28:34+01:00" level=fatal msg="unknown long flag '--collectors.exchange.list'\n" source="exporter.go:312"
```
The corresponding config file fields are ignored.

Version: master (https://github.com/prometheus-community/windows_exporter/commit/f623c0ed89903413c6fd7cc8ab2d447f3d20ea36)

**Comment by carlpett** [comment]
Thanks @fischerman!
@breed808 I think we were a bit hasty in merging #638 :( When we changed from registering on the global `kingpin` app to the specific one, we also made the collector flags not become registered.

**Comment by breed808** [comment]
Yes, the merge was a bit hasty as I hadn't tested the collector-specific flags.
We'll need to pass the kingpin `app` to the `collector` package in some manner.

**Comment by breed808** [comment]
I believe I've written a fix that resolves this, I'll create a PR once #651 is merged.
